storage/plugins/book-club/src/Repo.php

```
@@ -934,7 +945,22 @@ public function searchCatalogById(int $libroId): ?array
 
     public function deleteClubBook(int $clubBookId): bool
     {
-        return $this->exec('DELETE FROM bookclub_books WHERE id = ?', 'i', [$clubBookId]);
+        // Capture the external link BEFORE deleting the club-book row: deleting
+        // the child bookclub_books row does not cascade up to its parent
+        // bookclub_external_books, so a never-acquired external proposal would
+        // otherwise leave an orphaned metadata row behind.
+        $row = $this->row('SELECT external_book_id FROM bookclub_books WHERE id = ?', 'i', [$clubBookId]);
+        $ok = $this->exec('DELETE FROM bookclub_books WHERE id = ?', 'i', [$clubBookId]);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)